src/install/ddocWrapper.js

```diff
@@ -15,6 +15,24 @@ module.exports = (ddoc, mode) => {
 
   const appNotAlreadyUnzipped = app => !fs.existsSync(app.deployPath());
 
+  const appNotCurrent = app => {
+    const getCurrentPath = (app) => {
+      const currentPath = app.deployPath('current');
+      if (!fs.existsSync(currentPath)) {
+        return;
+      }
+      const linkString = fs.readlinkSync(currentPath);
+      if(!fs.existsSync(linkString)) {
+        return;
+      }
+
+      return linkString;
+    };
+
+    const currentPath = getCurrentPath(app);
+    return !currentPath || currentPath !== app.deployPath();
+  };
```

Comment:
I think it would be clearer if the `getCurrentPath` function was inlined, eg:

```js
const appNotCurrent = app => {
  const currentPath = app.deployPath('current');
  if (!fs.existsSync(currentPath)) {
    return false;
  }
  const linkString = fs.readlinkSync(currentPath);
  if (!fs.existsSync(linkString)) {
    return false;
  }
  return currentPath !== app.deployPath();
}
```

Reply:
Done.